content/2-2.md

```diff
@@ -125,31 +125,32 @@ Optional ideas to support assessment.
-* convey other states that menu items can have using color, shape, or icons
-* summarize related requirements for authors and designers to provide menu texts and styles
+* explain how appropriate CSS styles make menus easier to identify and understand, and more usable for people with disabilities using adaptive strategies such as text resize and customization, screen magnification, and touch screens
+* code styles to make menus appear as needed, for example vertical, horizontal, or as breadcrumb trails
+* mark up the state of the current menu item using the WAI-ARIA property `aria-current="page"`
+* convey other states of menu items using the appropriate semantics as well as color, shape, and styles
+* recite related requirements for authors and designers to provide appropriate menu texts and styles
 
 #### Teaching Ideas for Topic
 
-Optional ideas to teach the learning outcomes.
+Optional ideas to teach the learning outcomes:
 
 * Show examples of different types of menus, such as navigational, application, or breadcrumb trails. Explain that they need to be coded so that they are placed in the expected position within the page. Mention that menu location is a shared responsibility among different team members: content authors, designers, and developers. Examples of the expected positions of menus within the page are provided in the WAI tutorials on [Location](https://www.w3.org/WAI/tutorials/menus/styling/#location).
 * Introduce the different states that a menu item can have, such as hover, focus, current, active, or visited. Explain how to code these states so that they are perceived by all users. Examples of how to code these states are provided in the WAI tutorials, [Menu Items](https://www.w3.org/WAI/tutorials/menus/styling/#menu-items).
 * Show examples of different layouts for menus. Explain that menus should adapt to different text sizes users may choose. Mention that menus should have enough space so that they can be operated by all users. Considerations about readability and size of menu items are provided in the WAI tutorials, [General considerations](https://www.w3.org/WAI/tutorials/menus/styling/#general-considerations).
 
 #### Ideas to Assess Knowledge for Topic
 
-Optional ideas to support assessment.
+Optional ideas to support assessment:
 
-* Practical &mdash; Students are presented with different types of menus and are asked to code them so that they are placed where expected. Assess students knowledge of the menu location within web pages.
-* Practical &mdash; Students are asked to code the different states of menu items and to make them distinguishable from each other. Assess students knowledge of WAI-ARIA and CSS techniques to convey the states of menu items.
+* Practical &mdash; Students are presented with different types of menus and are asked to code them so that they are placed where expected. Assess students' knowledge of the menu location within web pages.
```

Comment:
I am the last person to ask about apostrophe placement but have developed many ways to avoid using them. Would "Assess student knowledge of the menu location within web pages." fit the bill and avoid people trying to workout if we have made an apostrophe catastrophe?

Reply:
This affects the whole resource as well. Tricky one for me also. In my Spanish brain this is "the knowledge of the students" (with no Saxon genitive at all as we don't have such structures).

I used "Students' knowledge" assuming "students" is plural and only needs apostrophe (no s after it). We hope this guidance supports the creation of courses that assess the knowledge of several students, not the knowledge of a specific student.

Not quite sure about what to do with this one. Maybe @slhenry can help with this. What about "students knowledge" (no apostrophe at all, but still plural)?

Reply:
My money is on @slhenry knowing precisely what to do here :)

Reply:
"Assess students' knowledge" is correct -- plural possessive.

I appreciate Kevin's approach of trying to limit apostrophes. I think in this case we want to keep it plural. So they'll just have to trust us that it's correct. :-)